src/server/components/lists/find/handlers/result.ts

```diff
@@ -0,0 +1,27 @@
+import { searchLawyers } from "server/components/lists/searches/lawyers";
+import { removeQueryParameter } from "../../helpers";
+import type { Request, Response } from "express";
+import { searchFuneralDirectors } from "server/components/lists/searches/funeral-directors";
+import { HttpException } from "server/middlewares/error-handlers";
+import { searchTranslatorsInterpreters } from "server/components/lists/searches/translators-interpreters";
+
+export async function get(req: Request, res: Response) {
+  const { serviceType } = req.params;
+
+  res.locals.answers = req.session.answers;
```

Comment:
Out of curiosity what is the benefit of adding this data to locals if it is already in the session?

Reply:
res.locals is where you want to store variables across the lifetime of a req/res, res.locals are also available in the render context. 

So instead of passing in a huge object to the render context, res.render("some-template, {...DEFAULTS, something, somethingElse, answers: req.session.answers}), I can set res.locals. 

It's especially useful if you're using middleware. You'll notice I've removed all of the getCsrfTokens from all of the res.renders - since I do that with `loadCsrfTokenIntoLocals`. 

session is where you want to store stuff for the lifetime of the session